src/composables/ViewRoutingSlip/usePaymentInformation.ts

```diff
@@ -42,6 +43,7 @@ export default function usePaymentInformation (_, context) {
       paymentIndex: paymentIndex
     }
     updateRoutingSlipChequeNumber(chequeNumToChange)
+    hasChequeNumberChanged.value = true
```

Comment:
Maybe compare to see if it's changed in the old routingSlip.value?

Otherwise one could change it, change it back.. and then it states it's changed?